generators/app/templates/.gitignore

```diff
@@ -1,3 +1,35 @@
+################################################
+# macOS
```

Comment:
There is a lot of useful stuff to include by default for a Craft project, but that would probably best be handled by modifying this file within a separate Craft task.